back/app/services/side_fx_app_configuration_service.rb

```diff
@@ -5,19 +5,9 @@ class SideFxAppConfigurationService
 
   def before_create(app_config, current_user = nil) end
 
-  def after_create(app_config, _ = nil)
-    return unless app_config.homepage_info_multiloc
+  def after_create(app_config, _ = nil) end
 
-    app_config.update!(
-      homepage_info_multiloc: TextImageService.new.swap_data_images(app_config, :homepage_info_multiloc)
-    )
-  end
-
-  def before_update(app_config, _ = nil)
-    return unless app_config.homepage_info_multiloc
-
-    app_config.homepage_info_multiloc = TextImageService.new.swap_data_images app_config, :homepage_info_multiloc
-  end
+  def before_update(app_config, _ = nil) end
```

Comment:
I left these as no-ops, for now. I need to check where they are called from, as it might be more than one place in the code (and an alias might be used for the class name in the calling line.